src/sentry/receivers/outbox/control.py

```diff
@@ -62,3 +65,35 @@ def process_organization_integration_update(object_identifier: int, **kwds: Any)
     ) is None:
         return
     organization_integration  # Currently we do not sync any other organization integration changes, but if we did, you can use this variable.
+
+
+@receiver(process_control_outbox, sender=OutboxCategory.WEBHOOK_PROXY)
+def process_async_webhooks(payload: Any, region_name: str, **kwds: Any):
+    from sentry.models.outbox import ControlOutbox
+    from sentry.silo.client import RegionSiloClient
+    from sentry.types.region import RegionResolutionError, get_region_by_name
+
+    try:
+        region = get_region_by_name(name=region_name)
+    except RegionResolutionError as e:
+        logger.error("webhook_proxy.region_resolution_error", extra={"error": e})
+        return
+
+    try:
+        webhook_payload = ControlOutbox.get_webhook_payload_from_outbox(payload=payload)
+    except Exception as e:
+        logger.error("webhook_proxy.invalid_payload_error", extra={"error": e})
+        return
+
+    response = RegionSiloClient(region=region).request(
+        method=webhook_payload.method,
+        path=webhook_payload.path,
+        headers=webhook_payload.headers,
+        # We need to send the body as raw bytes to avoid interfering with webhook signatures
+        data=webhook_payload.body,
+        json=False,
+        raw_response=True,
+    )
```

Comment:
If the response from the region isn't a 200, we should also raise as the destination did not accept the outbox message, and it needs to be retried.